### ISSUE TITLE
A11y_AzureDataStudio_ConnectToAMySQLServer_Keyboard: Tooltips provided for "Views and More Action", "New Connection", "New Server Group" and "Show Active Connections" are not accessible using keyboard keys.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22677]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG2.1.1; #Linux; #Keyboard; 

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. Select the Connections icon (if not already selected) from left bar.
3. TAB to "New connection" present besides to 'Servers' and hit ENTER key.
4. Fill all the required details and TAB to "Connect" and hit ENTER key.
5. Server gets connected. TAB to "Views and More Action", "New Connection", "New Server Group" and "Show Active Connections".
6. Observe that Tooltips provided "Views and More Action", "New Connection", "New Server Group" and "Show Active Connections" are not accessible using keyboard keys.

### Actual Result:​
Tooltips provided for "Views and More Action", "New Connection", "New Server Group" and "Show Active Connections" are not accessible using keyboard keys.

**Similar issue is also observed in below scenario:
Issue 1: Notebooks - Create**

1. Launch Azure Data Studio application.
2. TAB to "File" and hit ENTER key. TAB to "New Notebook" and hit ENTER key.
3. TAB to "Collapse cells", "Clear Results", "Trusted" and "Run with parameter".
4. Navigate using TAB key and observe Tooltips provided for "Collapse cells", "Clear Results", "Trusted" and "Run with parameter" are not accessible using keyboard keys.

### Expected Result:​
Tooltips provided "Views and More Action", "New Connection", "New Server Group" and "Show Active Connections" should be accessible using keyboard keys.

### User Impact:
Keyboard users will not be able to get the information about these controls.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachments
[Video 3-30 at 12.37 (1).webm](https://user-images.githubusercontent.com/90901519/228757457-8c9125d2-468e-41c9-9e33-2ceaeeb66d12.webm)
[Video 3-30 at 12.37.webm](https://user-images.githubusercontent.com/90901519/228757482-52a2c512-0b6d-4c6b-9f4d-85bfdc357c3b.webm)

**Comment by alanrenmsft** [comment]
@PoojaNamde this is an external issue and we already have an issue tracking this in ADS: https://github.com/microsoft/azuredatastudio/issues/19566, please lower the severity to Sev3 as the main issue is a Sev 3.

**Comment by BhanuPrakashGit** [comment]
Lowered the severity to Sev3.
